src/utils/command-helpers.js

```diff
@@ -198,6 +202,34 @@ const normalizeConfig = (config) =>
     ? { ...config, build: omit(config.build, ['publish', 'publishOrigin']) }
     : config
 
+const DEBOUNCE_WAIT = 100
+
+const watchDebounced = async (target, { depth, onAdd = () => {}, onChange = () => {}, onUnlink = () => {} }) => {
+  const watcher = chokidar.watch(target, { depth, ignored: /node_modules/, ignoreInitial: true })
+
+  await pEvent(watcher, 'ready')
```

Comment:
`chokidar` can [emit `error` events](https://github.com/paulmillr/chokidar#methods--events). 
Not a blocker for this PR, but we might consider handling those.